appserver/concurrent/concurrent-impl/src/main/java/org/glassfish/concurrent/admin/ManagedExecutorServiceBaseManager.java

```diff
@@ -205,6 +208,7 @@ protected void setAttributesOnConfigBean(ManagedExecutorServiceBase managedExecu
         managedExecutorService.setEnabled(enabled);
         //Fix for GLASSFISH-21251
         managedExecutorService.setLongRunningTasks(longRunningTasks);
+        managedExecutorService.setUseForkJoinPool(useForkJoinPool);
```

Comment:
Knock-on from suggestion to move `ManagedExecutorService` specific config out of `ManagedExecutorServiceBase` interface, this would need adjusting.